src/os/ObjectStore.h

```diff
@@ -1001,6 +1003,18 @@ class ObjectStore {
       _op->op = OP_NOP;
       data.ops++;
     }
+    /**
+     * create
+     *
+     * Ensure the non-existance of an object in a collection.
```

Comment:
create an object that does not yet exist (behavior is undefined if the object already exists)